src/Microsoft.Health.Fhir.SqlServer/Features/Schema/SchemaUpgradeRunner.cs

```diff
@@ -50,7 +50,7 @@ public void ApplySchema(int version)
             _logger.LogInformation("Completed applying schema {version}", version);
         }
 
-        private static string GetMigrationScript(int version)
+        public static string GetMigrationScript(int version)
```

Comment:
Since this is used by more than just the `SchemaUpgradeRunner` now, this might make sense in a different file.

Reply:
addressed it 👍